mapie_v1/regression.py

```diff
@@ -834,43 +834,43 @@ def predict(
 
 class ConformalizedQuantileRegressor:
     """
-    A conformal quantile regression model that generates prediction intervals
-    using quantile regression as the base estimator.
+    A model that combines quantile regression with conformal prediction to
+    generate reliable prediction intervals with specified coverage levels.
 
-    This approach provides prediction intervals by leveraging
-    quantile predictions and applying conformal adjustments to ensure coverage.
+    The `ConformalizedQuantileRegressor` leverages quantile regression as its
+    base estimator to predict conditional quantiles of the target variable,
+    and applies conformal adjustments to ensure prediction intervals achieve
+    the desired confidence levels. This approach is particularly useful in
+    uncertainty quantification for regression tasks.
 
     Parameters
     ----------
-    estimator : RegressorMixin, default=QuantileRegressor()
-        The base quantile regression estimator used to generate point and
-        interval predictions.
-
-    confidence_level : Union[float, List[float]], default=0.9
+    estimator : Union[RegressorMixin, Pipeline,
+                    List[Union[RegressorMixin, Pipeline]]]
+        The base quantile regression model(s) used to estimate the target
+        quantiles.
+        - In `fit` mode (`prefit=False`), this should be a single quantile
+          regression estimator (e.g., `QuantileRegressor`) or a pipeline
+          combining preprocessing and regression.
+        - In `prefit` mode (`prefit=True`), this must be a list of three
```

Comment:
Let's clarify the predit=True explanations here:

1. What type of models are supported
2. What are the lower, upper and median quantiles (ie must be (1-confidence_level)/2, confidence_level/2, 0.5)